commitments/eigenda.go

```diff
@@ -1,39 +1,40 @@
 package commitments
 
-type CertEncodingCommitment byte
+type EigenDACommitmentType byte
 
 const (
-	CertV0 CertEncodingCommitment = 0
+	// EigenDA V1
+	CertV0 EigenDACommitmentType = iota
+	// EigenDA V2
+	CertV1
 )
 
-// OPCommitment is the binary representation of a commitment.
+// CertCommitment is the binary representation of a commitment.
 type CertCommitment interface {
-	CommitmentType() CertEncodingCommitment
+	CommitmentType() EigenDACommitmentType
 	Encode() []byte
 	Verify(input []byte) error
 }
 
-type CertCommitmentV0 []byte
-
-// NewV0CertCommitment creates a new commitment from the given input.
-func NewV0CertCommitment(input []byte) CertCommitmentV0 {
-	return CertCommitmentV0(input)
+type EigenDACommitment struct {
+	prefix EigenDACommitmentType
+	b      []byte
```

Comment:
Think we wanted to call this `EigenDAVersionedCert` in https://app.excalidraw.com/s/1XPZRMVbRNH/26HgEWKo0sT ?